.deploystack/test.yaml

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
**Suggestion:**
Since most Online Boutique maintainers are unfamiliar with DeployStack, could you please add a comment at the top of this file explaining its purpose?

Reply:
Added documentation